docusaurus/docs/reactnative/basics/troubleshooting.mdx

```diff
@@ -350,6 +350,12 @@ Reanimated plugin has to be listed last.
 :::
 
 ### Blank screen on Android
+:::caution
+
+This step is only required for older version of (Reanimated 2).
+If you are using version >=3.1.0, you can skip this step.
```

Comment:
why not >= 3.0.. or I think its simpler for the reader if this section states that it is only relevant to reanimated v2 and not any other version